xbmc/network/websocket/WebSocketV8.cpp

```diff
@@ -50,7 +50,8 @@ bool CWebSocketV8::Handshake(const char* data, size_t length, std::string &respo
 
   // The request must be GET
   value = header.getMethod();
-  if (value == NULL || strnicmp(value, WS_HTTP_METHOD, strlen(WS_HTTP_METHOD)) != 0)
+  if (value == NULL ||
```

Comment:
Also minor thing from my side, it could be then a „nullptr“, but not mandatory for me.
There and few other places.